src/clients/node/README.md

```diff
@@ -21,7 +21,7 @@ First, create a directory for your project and `cd` into the directory.
 Then, install the TigerBeetle client:
 
 ```console
-npm install tigerbeetle-node
+npm install tigerbeetle-node@latest
```

Comment:
If we encourage installing any version, I don't think we should encourage installing a symbolic version versus a tagged version since we require the client and server versions to line up.

Reply:
That's true, ideally, we should update the documents on each build to include the current tag.
I added `@latest` because otherwise, it would install version `0.12.52`, which is too old to run the samples on Windows.

Instead, should I update the `package.json` file for all samples?